projects/ccf-rui/src/app/shared/components/details-label/details-label.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+
+/**
+ * Reusable component that takes an array of strings and renders
+ * that array as a string, separated by commas, in the icon color.
+ */
+@Component({
+  selector: 'ccf-details-label',
+  templateUrl: './details-label.component.html',
+  styleUrls: ['./details-label.component.scss']
+})
+export class DetailsLabelComponent {
+  /**
```

Comment:
Add `@HostBinding('class') readonly clsName = 'ccf-details-label';` Check rotation-slider to see how it is done together with theming.